### PR TITLE
checkbox-style

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: styleguide.css [hunk 1]
```diff
@@ -11,7 +11,7 @@ html {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-smoothing: antialiased;
-    text-shadow: 1px 1px 1px rgba(0,0,255,0.004);
+    text-shadow: 0.1rem 0.1rem 0.1rem rgba(0,0,255,0.004);
```
Comment (vinyll) [inline styleguide.css]: `rgba(0,0,255,0.004);`
missing spaces after commas.

Why `0.004` is it visible at all or should we remove it?

File: styleguide.css [hunk 2]
```diff
@@ -227,7 +227,9 @@ li a {
     position: relative;
 }
 
-input {
+
+input,
+button{
```
Comment (vinyll) [inline styleguide.css]: missing space

File: styleguide.css [hunk 3]
```diff
@@ -246,33 +248,37 @@ input::placeholder {
...
-input[type=submit] {
+input[type=submit],
+button {
     text-align: center;
     padding: 2rem 4rem;
     cursor: pointer;
 }
-input[type=submit]:hover {
+input[type=submit]:hover,
+button:hover{
```
Comment (vinyll) [inline styleguide.css]: missing space

File: styleguide.css [hunk 4]
```diff
@@ -246,33 +248,37 @@ input::placeholder {
...
     background-color: #00283C;
     color: #fff;
 }
 input[type=submit],
-input[type=submit]:hover {
+input[type=submit]:hover,
+button,
+button:hover{
```
Comment (vinyll) [inline styleguide.css]: missing space

**Review by vinyll — CHANGES_REQUESTED** [review]

**Comment by vinyll** [comment]
Le commit est bien apparu dans la PR 👍

**Comment by johangiraud** [comment]
ah cool

File: styleguide.css [hunk 5]
```diff
@@ -11,7 +11,7 @@ html {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-smoothing: antialiased;
-    text-shadow: 1px 1px 1px rgba(0,0,255,0.004);
+    text-shadow: 0.1rem 0.1rem 0.1rem rgba(0, 0, 255, 0.004); /* well ok, I could only find this shitty source : https://www.webfwd.co.uk/blog/how-to-smooth-website-fonts-using-css3/. I'm pretty sure that wasn't my original source, but anyway, let's remove the line */
```
Comment (vinyll) [inline styleguide.css:14]: Delete this line then?

File: styleguide.css [hunk 6]
```diff
@@ -87,7 +87,7 @@ summary {
     font-family: 'vremena_grotesk';
     src: url('./fonts/vremenagroteskbolditalic-webfont.woff2') format('woff2'),
     url('./fonts/vremenagroteskbolditalic-webfont.woff') format('woff');
-    font-weight: 800;
+    font-weight:800;
```
Comment (vinyll) [inline styleguide.css:90]: add a space after `:`

File: styleguide.css [hunk 7]
```diff
@@ -246,33 +248,37 @@ input::placeholder {
...
     background-color: #fffff8;
     border: 0.1rem solid #bde513;
-    padding: 20px;
+    padding: 2rem;
     margin: 5rem 0;
 }
 
 #signin input[type=text],
 #signin input[type=email] {
     border:none;
```
Comment (vinyll) [inline styleguide.css:278]: add a space after `:`

File: styleguide.css [hunk 8]
```diff
@@ -246,33 +248,37 @@ input::placeholder {
...
     border-bottom: 0.1rem solid #bde513;
-    padding-left: 0;
     background-color: transparent;
+    padding:1rem 1rem 1rem 0;
```
Comment (vinyll) [inline styleguide.css:281]: add a space after `:`

File: styleguide.css [hunk 9]
```diff
@@ -283,10 +289,11 @@ form#signin {
 
 [type=checkbox]:not(:checked) + label,
 [type=checkbox]:checked + label {
+    display: block;
     position: relative;
-    padding-left: 35px;
+    padding-left: 3.5rem;
     cursor: pointer;
-    margin:50px 0;
+    margin:5rem 0;
```
Comment (vinyll) [inline styleguide.css:296]: add a space after `:`

File: styleguide.css [hunk 10]
```diff
@@ -283,10 +289,11 @@ form#signin {
 
 [type=checkbox]:not(:checked) + label,
 [type=checkbox]:checked + label {
+    display: block;
     position: relative;
-    padding-left: 35px;
+    padding-left: 3.5rem;
     cursor: pointer;
```
Comment (vinyll) [inline styleguide.css:295]: With a `pointer` cursor on labels, won't it pretend like it's a clickable element (button or link)?

File: styleguide.css [hunk 11]
```diff
@@ -308,10 +315,10 @@ form#signin {
     content: '\ea10';
     font-style: normal;
```
Comment (vinyll) [inline styleguide.css:316]: is this required? `normal` is the default for a label, right?

File: styleguide.css (hunk 10 above)
Comment (johangiraud) [reply styleguide.css:295]: it's on label[type=submit] only

File: styleguide.css (hunk 11 above)
Comment (johangiraud) [reply styleguide.css:316]: ok, not required then

File: styleguide.css (hunk 10 above)
Comment (vinyll) [reply styleguide.css:295]: Right, on a label that is attached to a checkbox only 👍 

HTML also permits you to write a checkbox inside of a label:

```
<label><input type=checkbox> Accept this?</label>
```

Comment (johangiraud) [reply styleguide.css:295]: ah oui, label which is after a checkbox.
ah ok, didn't know. Well if the `label` is after the `input` it's fine

**Comment by vinyll** [comment]
I'm available to clarify @cafebabel-webmaster on any point if needed ;)

**Pull request merged**